cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -128,13 +153,13 @@ public void tryConnect(ConnectionConfig connectionConfig, PrintStream output, bo
   }
 
   public void updateAccessToken(PrintStream output) throws IOException {
-    AccessToken newAccessToken = getNewAccessToken(clientConfig.getConnectionConfig(), output, false);
+    AccessToken newAccessToken = getNewAccessToken(clientConfig, output, false);
     clientConfig.setAccessToken(newAccessToken);
   }
 
   private void checkConnection(ClientConfig baseClientConfig,
-                               ConnectionConfig connectionInfo,
```

Comment:
Seems like this method should be more like "createClientConfig" that takes a ClientConfig.Builder and return a ClientConfig by setting the connectionConfig to the builder.